### PR TITLE
Job card states

### PR DESCRIPTION
**Pull request by PepeFranco**
Job card has now a hovered/active state with different background color.
Title now works as a link as it looks different if already visited.
As requested on https://trello.com/c/AdUar2Xi/517-styleguide-job-card-states

### CODE REVIEW

File: react/JobCard/JobCard.less [hunk 1]
```diff
@@ -63,7 +66,7 @@
 }
 
 .titleKeyword {
-  color: darken(@actionBlue, 15%);
+  color: darken(@actionBlue, 10%);
```
Comment (mengtzu) [inline react/JobCard/JobCard.less]: Not a blocker for this PR but I'm starting to wonder if to much darken/lighten/desaturate is an anti pattern and we should make those colours proper named variants themselves (like @actionBlueActive or whatever)

File: theme/palette/brand.less [hunk 2]
```diff
@@ -8,6 +8,7 @@
 // Seekasia brand colors
 /* stylelint-disable */
 @saBlue: #0D3880;
+@saVisitedLink: #6C488F;
```
Comment (mengtzu) [inline theme/palette/brand.less]: This probably belongs wherever `@hyperlink` is (`elements.less?`).

Comment (PepeFranco) [reply theme/palette/brand.less]: Noted and moved

**Comment by PepeFranco** [comment]
@mengtzu Forgot to add the hover state, added it now. Can you take a look?

**Pull request merged**